### PR TITLE
add chain with wallet, add buttons to switch chain

### PR DESCRIPTION
**Pull request by ismellpillows**

### CODE REVIEW

File: src/components/Navbar/WalletPopup.tsx [hunk 1]
```diff
@@ -16,6 +16,22 @@ const CONNECT_TEXT = 'MetaMask';
 const CONNECTING_TEXT = 'MetaMask (Connecting...)';
 const CONNECTED_TEXT = 'MetaMask Connected';
 
+const zkSyncChainParameter = {
+  chainId: 280,
+  chainName: 'zkSync Goerli Test Network',
+  nativeCurrency: {
+    name: "Goerli ETH",
```
Comment (natlou) [inline src/components/Navbar/WalletPopup.tsx]: Ethereum

File: src/components/Navbar/WalletPopup.tsx [hunk 2]
```diff
@@ -16,6 +16,22 @@ const CONNECT_TEXT = 'MetaMask';
...
+    symbol: "gorETH",
```
Comment (natlou) [inline src/components/Navbar/WalletPopup.tsx]: ETH

File: src/components/Navbar/WalletPopup.tsx [hunk 3]
```diff
@@ -16,6 +16,22 @@ const CONNECT_TEXT = 'MetaMask';
...
+  rpcUrls: ["https://zksync2-testnet.zksync.dev"],
+  blockExplorerUrls: ["https://zksync2-testnet.zkscan.io"],
+}
+
+export function connect() {
+  metaMask.activate(zkSyncChainParameter);
+}
+
 const WalletPopup = ({ setNetwork }: WalletPopupProps) => {
   const {
```
Comment (natlou) [inline src/components/Navbar/WalletPopup.tsx:32]: unsure what this does, let me check

File: src/components/SwapMenu/SwapButton.tsx [hunk 4]
```diff
@@ -16,13 +18,16 @@ const SwapButton = () => {
     setWalletPopupOpen(true);
   }
 
+  const handleClick = walletConnected ? (chainIsCorrect ? swap : connect) : openWalletPopup;
```
Comment (natlou) [inline src/components/SwapMenu/SwapButton.tsx:21]: good logic

File: src/components/SwapMenu/index.tsx [hunk 5]
```diff
@@ -166,7 +166,7 @@ const SwapMenu = () => {
             {Array.from(transactionInformation)
               .slice(1)
               .map(([label, value]) => (
-                <div className="flex place-content-between text-sm font-bold">
+                <div className="flex place-content-between text-sm font-bold" key={label}>
```
Comment (natlou) [inline src/components/SwapMenu/index.tsx:169]: choice of key is correct, good

File: src/components/Homepage/tutorial.tsx [hunk 6]
```diff
@@ -1,37 +1,20 @@
-import React, { useState, useEffect, useContext } from 'react';
-import Image from 'next/image';
+import React, { useContext } from 'react';
```
Comment (natlou) [inline src/components/Homepage/tutorial.tsx:1]: okay

File: src/components/Homepage/tutorial.tsx [hunk 7]
```diff
@@ -1,37 +1,20 @@
...
-import NetworksPopup from '../Navbar/NetworksPopup';
-import { WalletPopupContext } from '../../context/WalletPopupProvider';
-import { networks } from '../Navbar/NetworksPopup';
-import { hooks } from '../../connectors/metaMask';
-
-const { useIsActive } = hooks;
-
-
-
+import { PopupContext } from '../../context/PopupProvider';
```
Comment (natlou) [inline src/components/Homepage/tutorial.tsx:3]: makes sense

Comment (natlou) [reply src/components/Homepage/tutorial.tsx:3]: I dont know what should be under features and what should be under components

File: src/components/Navbar/WalletPopup.tsx (hunk 3 above)
Comment (natlou) [reply src/components/Navbar/WalletPopup.tsx:32]: i see

**Review by natlou — APPROVED** [review]

File: src/components/Navbar/WalletPopup.tsx (hunk 2 above)
Comment (ismellpillows) [reply src/components/Navbar/WalletPopup.tsx]: fixed

**Review by natlou — APPROVED** [review]

**Comment by natlou** [comment]
They all look good

**Pull request merged**